ivadomed/main.py

```diff
@@ -295,6 +295,10 @@ def run_segment_command(context, model_params):
                 pred_path.mkdir(parents=True)
 
             for pred, target in zip(pred_list, target_list):
+                # Set the target to first element if the target is a list (e.g. multi-rater setting)
+                if isinstance(target, list):
+                    target = target[0]
```

Comment:
The `target_list` is also used below at line 308 for microscopy (`pred_to_png`) and would also fail in case of multiple raters.
So whichever implementation we do to update the target list, I would do it before the for loop at 297. This way, we could pass the updated list to both the for loop and the `pred_to_png` function.